Watching for file changes with StatReloader
Not Found: /favicon.ico
Internal Server Error: /admin/login/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 444, in login
    return LoginView.as_view(**defaults)(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/decorators/debug.py", line 143, in sensitive_post_parameters_wrapper
    return view(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/views.py", line 89, in dispatch
    return super().dispatch(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/generic/base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/generic/edit.py", line 150, in post
    if form.is_valid():
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 197, in is_valid
    return self.is_bound and not self.errors
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 192, in errors
    self.full_clean()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 326, in full_clean
    self._clean_form()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 342, in _clean_form
    cleaned_data = self.clean()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/forms.py", line 362, in clean
    self.confirm_login_allowed(self.user_cache)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/forms.py", line 22, in confirm_login_allowed
    if not user.is_staff:
AttributeError: 'SuperUser' object has no attribute 'is_staff'
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 241, in inner
    return view(request, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 557, in index
    app_list = self.get_app_list(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 541, in get_app_list
    app_dict = self._build_app_dict(request, app_label)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/sites.py", line 483, in _build_app_dict
    has_module_perms = model_admin.has_module_permission(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/admin/options.py", line 640, in has_module_permission
    return request.user.has_module_perms(self.opts.app_label)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/utils/functional.py", line 253, in inner
    return func(_wrapped, *args)
AttributeError: 'SuperUser' object has no attribute 'has_module_perms'
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Using selector: EpollSelector
Using selector: EpollSelector
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /login/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 15, in login_view
    user = form.clean()
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py", line 28, in clean
    cleaned_data = super().clean()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 363, in clean
    return self.cleaned_data
AttributeError: 'CustomAuthenticationForm' object has no attribute 'cleaned_data'
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /login/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 15, in login_view
    user = form.auth()
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py", line 28, in auth
    cleaned_data = super().clean()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/forms/forms.py", line 363, in clean
    return self.cleaned_data
AttributeError: 'CustomAuthenticationForm' object has no attribute 'cleaned_data'
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /login/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 16, in login_view
    user = form.get_user()
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py", line 49, in get_user
    return self.cleaned_data.get('user')
AttributeError: 'str' object has no attribute 'get'
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/settings.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/forms.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
<<<<<<< HEAD
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
=======
Internal Server Error: /changepassword/
Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py", line 71, in changepassword
    return render(request, 'app/changepassword.html', context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: app/changepassword.html
Not Found: /wait.html
Not Found: /templates/app/wait.html
Not Found: /forgot-password
Not Found: /forgot-password
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /
Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: nmcnpmapp_superuser

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py", line 42, in homepage
    return render(request, 'app/homepage.html', {'articles': articles})
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\backends\django.py", line 107, in render
    return self.template.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 171, in render
    return self._render(context)
           ~~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 163, in _render
    return self.nodelist.render(context)
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
                               ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 969, in render_annotated
    return self.render(context)
           ~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\defaulttags.py", line 320, in render
    match = condition.eval(context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\defaulttags.py", line 886, in eval
    return self.value.resolve(context, ignore_failures=True)
           ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 718, in resolve
    obj = self.var.resolve(context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 850, in resolve
    value = self._resolve_lookup(context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 883, in _resolve_lookup
    current = current[bit]
              ~~~~~~~^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\utils\functional.py", line 251, in inner
    self._setup()
    ~~~~~~~~~~~^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\utils\functional.py", line 404, in _setup
    self._wrapped = self._setupfunc()
                    ~~~~~~~~~~~~~~~^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\middleware.py", line 37, in <lambda>
    request.user = SimpleLazyObject(lambda: get_user(request))
                                            ~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\middleware.py", line 17, in get_user
    request._cached_user = auth.get_user(request)
                           ~~~~~~~~~~~~~^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\__init__.py", line 216, in get_user
    user = backend.get_user(user_id)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\backends.py", line 157, in get_user
    user = UserModel._default_manager.get(pk=user_id)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\query.py", line 645, in get
    num = len(clone)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
    ~~~~~~~~~~~~~~~^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
        chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size
    )
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        sql, params, many=False, executor=self._execute
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: nmcnpmapp_superuser
Not Found: /favicon.ico
>>>>>>> 804cd5416c159f2e5f7cae9f9b3ae8a69660a3bd
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
<<<<<<< HEAD
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
=======
Not Found: /favicon.ico
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\forms.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\forms.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py changed, reloading.
Watching for file changes with StatReloader
<<<<<<< HEAD
>>>>>>> 804cd5416c159f2e5f7cae9f9b3ae8a69660a3bd
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Internal Server Error: /admin/nmcnpmapp/charge/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Internal Server Error: /admin/nmcnpmapp/roomuser/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Internal Server Error: /admin/nmcnpmapp/roomuser/1/change/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Watching for file changes with StatReloader
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/familymember/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such column: nmcnpmapp_familymember.date_of_birth

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2140, in changelist_view
    "selection_note": _("0 of %(cnt)s selected") % {"cnt": len(cl.result_list)},
                                                           ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\compiler.py", line 1574, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\backends\sqlite3\base.py", line 354, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such column: nmcnpmapp_familymember.date_of_birth
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Internal Server Error: /admin/nmcnpmapp/familymember/add/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 839, in get_form
    return modelform_factory(self.model, **defaults)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\forms\models.py", line 654, in modelform_factory
    return type(form)(class_name, (form,), form_class_attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\forms\models.py", line 334, in __new__
    raise FieldError(message)
django.core.exceptions.FieldError: Unknown field(s) (age) specified for FamilyMember

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1961, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1820, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1855, in _changeform_view
    ModelForm = self.get_form(
                ^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 841, in get_form
    raise FieldError(
django.core.exceptions.FieldError: Unknown field(s) (age) specified for FamilyMember. Check fields/fieldsets/exclude attributes of class FamilyMemberAdmin.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
=======
Watching for file changes with StatReloader
Not Found: /favicon.ico
>>>>>>> 30b3a9c3abf656a55c014543cc8bdd2f6f38c691
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Not Found: /favicon.ico
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/roomuser/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Internal Server Error: /admin/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 260, in wrapper
    return self.admin_view(view, cacheable)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 229, in inner
    if not self.has_permission(request):
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 202, in has_permission
    return request.user.is_active and request.user.is_staff
                                      ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\functional.py", line 253, in inner
    return func(_wrapped, *args)
           ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'RoomUser' object has no attribute 'is_staff'
Not Found: /favicon.ico
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /admin/nmcnpmapp/apartment/
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Not Found: /favicon.ico
Internal Server Error: /admin/nmcnpmapp/vehicle/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/vehicle/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/payment/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/charge/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Internal Server Error: /admin/nmcnpmapp/charge/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/charge/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /admin/nmcnpmapp/charge/
Traceback (most recent call last):
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 718, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 2001, in changelist_view
    cl = self.get_changelist_instance(request)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 866, in get_changelist_instance
    return ChangeList(
           ^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 145, in __init__
    self.queryset = self.get_queryset(request)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\views\main.py", line 583, in get_queryset
    qs, search_may_have_duplicates = self.model_admin.get_search_results(
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\contrib\admin\options.py", line 1222, in get_search_results
    queryset = queryset.filter(models.Q.create(term_queries))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1476, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1494, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\query.py", line 1501, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1609, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1468, in build_filter
    return self._add_q(
           ^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1641, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1555, in build_filter
    condition = self.build_lookup(lookups, col, value)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1379, in build_lookup
    lhs = self.try_transform(lhs, lookup_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\admin\.conda\envs\software\Lib\site-packages\django\db\models\sql\query.py", line 1423, in try_transform
    raise FieldError(
django.core.exceptions.FieldError: Unsupported lookup 'icontains' for ForeignKey or join on the field not permitted.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
